src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -7,4 +7,5 @@
  <sessions-enabled>true</sessions-enabled>
  <runtime>java8</runtime>
  <resource-files><include path="/**.csv" /></resource-files>
+ <resource-files><include path="/**.json" /></resource-files>
```

Comment:
should we just use one resource-files tag as it was causing problem while deploying ?